content/en/docs/test.md

```diff
@@ -0,0 +1,248 @@
+  - To make a sub-item, indent two spaces.
+    - This is a sub-sub-item.
+
+
+
+- This is a new list. It has two blank lines from the previous list, If it only
+  had one blank line, it would still show up as part of the previous list.
+- Bullet lists can have paragraphs or block elements within them.
+
+  Just indent the content to be even with the text of the bullet point, rather
+  than the bullet itself.
+
+  ```bash
+  $ ls -l
+  ```
+  
+  - And a sub-list after some block-level content
+- A bullet list item can contain a numbered list.
+  1.  Numbered sub-list item 1
+  2.  Numbered sub-list item 2
+
+### Numbered lists
+
+1.  This is a list item
+2.  This is another list item in the same list. The number you use in Markdown
+    does not necessarily correlate to the number in the final output. By
+    convention, we keep them in sync.
+3.  For single-digit numbered lists, using two spaces after the period makes
+    interior block-level content line up better along tab-stops.
+
+
+
+1.  This is a new list. It has two blank lines from the previous list, If it only
+    had one blank line, it would still show up as part of the previous list.
+2.  Numbered lists can have paragraphs or block elements within them.
+
+    Just indent the content to be even with the text of the bullet point, rather
+    than the bullet itself.
+
+    ```bash
+    $ ls -l
+    ```
+  
+  - And a sub-list after some block-level content
+
+
+### Checklists
+
+- [ ] This is a checklist item
+- [x] This is a selected checklist item
+
+## Code blocks
+
+You can create code blocks two different ways:
+
+- by indenting the entire code block 4 spaces from the content
+
+      this is a code block created by indentation
+      
+- by surrounding the code block with three back-tick characters on lines before
+  and after the code block.
+  
+  ```
+  this is a code block created by back-ticks
+  ```
+
+The back-tick method has some advantages.
+
+- It is more compact when viewing the source code.
+- It allows you to specify what language the code block is in, for syntax
+  highlighting.
+- It has a definite ending. Sometimes, the indentation method breaks with
+  languages where spacing is significant, like Python or YAML.
+
+To specify the language for the code block, put it directly after the first
+grouping of back-ticks:
+
+```bash
+$ ls -l
+```
+
+## Links
+
+To format a link, put the link text inside square brackets, followed by the
+link target in parentheses. [Link to Kubernetes.io](https://kubernetes.io/) or
+[Relative link to Kubernetes.io](/)
+
+You can also use HTML, but it is not preferred.
+<a href="https://kubernetes.io/">Link to Kubernetes.io</a>
+
+
+## Images
+
+To format an image, use similar syntax to [links](#links), but add a leading `!`
+character. The square brackets contain the image's alt text. Try to always use
+alt text so that people using screen readers can get some benefit from the
+image.
+
+![pencil icon](static/images/pencil.png)
```

Comment:
None of these pencil icons seem to be showing up. I think `/static/*` needs to be moved into `/content/en/`.

Reply:
I'm not sure what I need to do in this PR about the images...